tensorflow/core/kernels/mkl_pooling_ops_common.h

```diff
@@ -163,6 +477,43 @@ class MklPoolingOpBase : public OpKernel {
     }
   }
 
+  void PoolParamsToDims(MklPoolParameters* pool_params,
+                        memory::dims* filter_dims,
+                        memory::dims* strides,
+                        memory::dims* padding_left,
+                        memory::dims* padding_right) {
+    *filter_dims = {pool_params->window_rows, pool_params->window_cols};
+    *strides = {pool_params->row_stride, pool_params->col_stride};
+    *padding_left = {static_cast<int>(pool_params->pad_top),
+                     static_cast<int>(pool_params->pad_left)};
+    *padding_right = {static_cast<int>(pool_params->pad_bottom),
+                      static_cast<int>(pool_params->pad_right)};
+  }
+
+  void AllocateEmptyOutputTensor(
```

Comment:
I think it would make sense to move this to the base class. I suspect we might want to use this shorthand in a number of other kernels.

Reply:
Well, it does not make sense to move this to opkernel base class as it involves "pooling" specific parameters. 
This method does not apply to other MKL ops.